### PR TITLE
Seori/DSODA-97

### PR DESCRIPTION
**Pull request by msbeethoven**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -32,5 +32,8 @@ module.exports = {
         node: true
       }
     }
-  ]
-};
+  ],
+  globals: {
+    "Promise": true
+  }
+};
```
Comment (bkammin) [inline .eslintrc.js]: Why is this added? We should avoid global references when possible.

File: app/components/nypr-o-footer/component.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
...
 export default Component.extend({
   layout,
   tagName: '',
+  NEWSLETTER_ENDPOINT: `${config.optInService}/v1/subscribe/mailchimp`,
+  NEWSLETTER_PARAMS: {list: config.jukeboxNewsletter}
+
+
   // calculatePosition(trigger, content) {
   //   return positionBottomCenter(trigger, content, 5);
   // }
```
Comment (bkammin) [inline app/components/nypr-o-footer/component.js]: delete commented out code if it is no longer needed.

File: app/components/nypr-o-footer/component.js [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
-// import { positionBottomCenter } from '@nypr-design-system' ;
+import config from 'nypr-jukeboxes/config/environment';
 
 export default Component.extend({
   layout,
   tagName: '',
+  NEWSLETTER_ENDPOINT: `${config.optInService}/v1/subscribe/mailchimp`,
+  NEWSLETTER_PARAMS: {list: config.jukeboxNewsletter}
+
+
```
Comment (bkammin) [inline app/components/nypr-o-footer/component.js]: too many newlines

File: app/nypr-m-newsletter-form.js [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+import Component from '@ember/component';
+import layout from '../templates/components/nypr-m-newsletter-form';
+
+import { computed } from '@ember/object';
+import { assert } from '@ember/debug';
+import { task } from 'ember-concurrency';
+
+import fetch from 'fetch';
+
+export const DEFAULT_LEGAL = "By submitting y=our information, you're agreeing to receive communications from New York Public Radio in accordance with our Terms."
```
Comment (bkammin) [inline app/nypr-m-newsletter-form.js]: Delete `app/nypr-m-newsletter-form.js` from this PR and use the `nypr-design-system` version, because it looks like the only change to this file from the `nypr-design-system` version is a typo:

```
-export const DEFAULT_LEGAL = "By submitting y=our information, you're agreeing to receive communications from New York Public Radio in accordance with our Terms."
+export const DEFAULT_LEGAL = "By submitting your information, you're agreeing to receive communications from New York Public Radio in accordance with our Terms."
```

File: app/components/nypr-o-footer/component.js (hunk 2 above)
Comment (msbeethoven) [reply app/components/nypr-o-footer/component.js]: I think this was left by someone else? They may circle back around to it? But deleted either way.

File: .eslintrc.js (hunk 1 above)
Comment (bkammin) [reply .eslintrc.js]: Add a newline after the last line of the file. If you do that, `.eslintrc.js` will no longer show up in this PR and things will be a bit cleaner.

File: app/components/nypr-o-footer/component.js [hunk 5]
```diff
@@ -1,11 +1,10 @@
...
 
 export default Component.extend({
   layout,
   tagName: '',
-  // calculatePosition(trigger, content) {
-  //   return positionBottomCenter(trigger, content, 5);
-  // }
+  NEWSLETTER_ENDPOINT: `${config.optInService}/v1/subscribe/mailchimp`,
+  NEWSLETTER_PARAMS: {list: config.jukeboxNewsletter}
 });
```
Comment (bkammin) [inline app/components/nypr-o-footer/component.js]: Add a newline at the end of file.

File: app/controllers/application.js [hunk 6]
```diff
@@ -5,6 +5,7 @@ import { computed } from '@ember/object';
 import { reads } from '@ember/object/computed';
 import move from 'ember-animated/motions/move';
 import { easeInAndOut } from 'ember-animated/easings/cosine';
+import config from '../config/environment'
```
Comment (bkammin) [inline app/controllers/application.js]: Remove `config`, it's no longer needed.

File: app/nypr-m-newsletter-form.js (hunk 4 above)
Comment (bkammin) [reply app/nypr-m-newsletter-form.js]: You should be able to delete the entire `app/nypr-m-newsletter-form.js` from this PR.

File: .eslintrc.js (hunk 1 above)
Comment (msbeethoven) [reply .eslintrc.js]: Strange -- thought I did these changes from the first PR -- maybe I rewrote over them when there was a conflict. Either way,  taken these changes!
